verification/curator-service/api/src/controllers/cases.ts

```diff
@@ -152,6 +160,11 @@ export default class CasesController {
         }
     };
 
+    /**
+     * udpate will try to geocode the location found in the request if needed
```

Comment:
upsert

But if you wanted, you could probably just start with "Geocodes the [...]" since the function name is clear. Including the name is fine by me, especially if it's conventional in tsdoc.

Reply:
Ah sorry that's my Golang, all comments have to start with the name of the thing that they are commenting.
That's because godoc (a cli to generate and extract docs) becomes very useful when you do this (https://blog.golang.org/godoc).
As I did the same for all other methods I just changed to upsert for consistency, thanks for catching it.

Reply:
Plus, this way if the code ever gets rewritten in Go, we won't even have to change the comments.

Reply:
don't tempt me!! ;-) I got more than enough time left in this fellowhsip to do it!!!